src/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -0,0 +1,92 @@
+"""
+    NormInfinityBridge{T}
+
+The `NormInfinityCone` is representable with LP constraints, since ``t \\ge \\max_i |x_i|`` if and only if ``t \\ge x_i`` and ``t \\ge -x_i`` for all ``i``.
+"""
+struct NormInfinityBridge{T} <: AbstractBridge
+    ge_index::CI{MOI.VectorAffineFunction{T}, MOI.Nonnegatives}
+end
+function bridge_constraint(::Type{NormInfinityBridge{T}}, model, f::MOI.VectorOfVariables, s::MOI.NormInfinityCone) where T
+    return bridge_constraint(NormInfinityBridge{T}, model, MOI.VectorAffineFunction{T}(f), s)
```

Comment:
You shouldn't need this if you use `MOIU.operate`, it should convert when needed. The bridges that uses this were written before `MOIU.operate`